### ISSUE TITLE
mySIMBL doesn't automatically load into all applications

### ISSUE THREAD

**Issue by avncharlie** [w0lfschild/mySIMBL#25]
I use Flashlight, which uses mySIMBL. Flashlight only works if I click "Load into all applications" in mySIMBL every startup. 

Is there any way to automatically load into all applications at login? It can get quite tedious having to open mySIMBL and do this every startup.

Flashlight is a helpful app that I find extremely useful. Thank you for your continued work into these applications.

**Comment by w0lfschild** [comment]
What version of mySIMBL are you using? Is everything up to date?

**Comment by avncharlie** [comment]
Everything is up to date. I'm using version 0.5.3.

**Comment by avncharlie** [comment]
I recently changed something on my startup items in System Preferences, and it started to work. I don't recall what exactly I did. There is a warning next to the SIMBLHelper application in the startup items, however, it still seems to work.

If it helps, here is a screenshot of my startup items.
![screen shot 2017-02-03 at 8 18 21 am](https://cloud.githubusercontent.com/assets/24058782/22568846/6471a71c-e9e9-11e6-9514-0a255bce1ebe.png)

Because it works, I will close this issue now. 

Thank you.
